packages/nodejs/src/typeTranslation.ts

```diff
@@ -0,0 +1,282 @@
+import * as v1 from '@concordium/common-sdk';
+import * as v2 from '../grpc/v2/concordium/types';
+import { recordMap, unwrap } from './util';
+
+function unwrapToHex(x: Uint8Array | undefined): string {
+    return Buffer.from(unwrap(x)).toString('hex');
+}
+
+function unwrapValueToHex(x: v2.EncryptedAmount | v2.TransactionHash): string {
+    return unwrapToHex(x.value);
+}
+
+function transRelease(release: v2.Release): v1.ReleaseScheduleWithTransactions {
+    return {
+        timestamp: new Date(Number(unwrap(release.timestamp?.value)) * 1000),
```

Comment:
```suggestion
        timestamp: transTimestamp(release.timestamp),
```
Any reason not to use that helper? :upside_down_face: